declarations/Qwen Chat.json

```
@@ -15,7 +15,7 @@
       "fetch": "https://chat.qwen.ai/legal-agreement/privacy-policy",
       "executeClientScripts": true,
       "select": [
-        ".legalPage"
+        ".app"
```

Will work, but might be more stable with:

```suggestion
        ".privacy-legal-page"
```